applications/dashboard/src/scripts/dom.ts

```diff
@@ -319,3 +321,51 @@ export function ensureScript(scriptUrl: string) {
         }
     });
 }
+
+export function getNextTabbableElement(
```

Comment:
A utility function like this should be added with unit tests and shouldn't log anything to the console. We can probably just kick this down the road until we go to resolve the focus issues on mobile.